testing/run_tests.py

```diff
@@ -752,6 +753,12 @@ def run_objc_tests(ios_variant='ios_debug_sim_unopt', test_filter=None):
     ios_unit_test_dir = os.path.join(
         BUILDROOT_DIR, 'flutter', 'testing', 'ios', 'IosUnitTests'
     )
+
+    result_bundle_temp = tempfile.TemporaryDirectory(
```

Comment:
The way you use `TemporaryDirectory` is with a `with` statement to make sure that it gets cleaned up.